src/cmds/qsub.c

```diff
@@ -4440,6 +4440,11 @@ do_connect(char *server_out, char *retmsg)
 		return (pbs_errno);
 	}
 
+	if (pbs_connection_set_nodelay(sd_svr) == -1) {
```

Comment:
Calling of the function pbs_connection_set_nodelay here is not required because we already call this function inside cnt2server_extend.